src/back/controllers/api/routes/categories/categories.js

```diff
@@ -1,19 +1,44 @@
 'use strict';
 
-const category = require('./../../models/category');
-const user = require('./../../models/user');
+const categoryModel = require('./../../models/category');
+const userModel = require('./../../models/user');
 
 const getAll = (req, res) => {
-    category.find({}, (err, categories) => {
-        res.json(categories);
+    categoryModel.find({}, (err, findedCategories) => {
```

Comment:
please extract all business logic far from controller eg route handler,
route handler just bridge between our application boundaries and rest api protocol